assets/js/app/LocusZoom.js

```diff
@@ -52,6 +65,52 @@
         return "" + (p / Math.pow(10, 6)).toFixed(places);
     };
 
+    //parse numbers like 5Mb and 1.4kB 
+    function parsePosition (x) {
+        var val = x.toUpperCase();
+        val = val.replace(",","");
+        var suffixre = /([KMG])[B]*$/;
+        var suffix = suffixre.exec(val);
+        var mult = 1;
+        if (suffix) {
+            if (suffix[1]=="M") {
+                mult = 1e6;
+            } else if (suffix[1]=="G") {
+                mult = 1e9;
+            } else {
+                mult = 1e3; //K
+            }
+            val = val.replace(suffixre,"");
+        }
+        val = Number(val) * mult;
+        return val;
+    }
+
+    // Parse region queries that look like
+    // chr:start-top
```

Comment:
Edit: just remembered that **end** is what we've settled on as opposed to **stop**